bundles/EcommerceFrameworkBundle/FilterService/FilterType/ElasticSearch/MultiSelect.php

```diff
@@ -68,20 +72,23 @@ public function addCondition(AbstractFilterDefinitionType $filterDefinition, Pro
                 }
             }
 
-            $attributeConfig = $productList->getTenantConfig()->getAttributeConfig()[$field];
-            if ($attributeConfig['type'] == 'boolean') {
-                foreach ($quotedValues as $k => $v) {
-                    $quotedValues[$k] = (bool)$v;
+            if (method_exists($productList, 'getTenantConfig')) {
```

Comment:
if should only cover first sub-if

```suggestion
            if (method_exists($productList, 'getTenantConfig')) {
                $config = $productList->getTenantConfig();
                $attributeConfig = $config->getAttributeConfig()[$field];
                if ($attributeConfig['type'] == 'boolean') {
                    foreach ($quotedValues as $k => $v) {
                        $quotedValues[$k] = (bool)$v;
                    }
                }
            }
```